### ISSUE TITLE
Accessibility - Step 7 Topic 1 Screen - "Example In Action" Text has Poor Color Contrast

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#60]
On the Step 7 Topic 1 screen the color contrast of the  "Example In Action" text has poor color contrast. The light blue text on a light gray background does not have enough contrast.

The color contrast ratio needs to be 4.5:1 or higher to pass.

The color contrast issue will apply to any other screens that have a similar color scheme.

**Comment by adriennerlong** [comment]
John, if we change the light blue to a darker blue (similar to the dark blue used for hyperlinks) will that pass? Something like this: 
![example in action_new color](https://user-images.githubusercontent.com/31417858/29983487-1df917a8-8f24-11e7-900a-23de28a0f0b6.PNG)

**Comment by jfc3-dol** [comment]
@adriennerlong,

Yes, the darker blue on the gray background passes with enough color contrast.

Is the gray background darker too? It seems to be to me but it could be because the the darker blue text.

**Comment by adriennerlong** [comment]
Yes,  that grey background is darker than what appears in the app. It was the closest option I had in Microsoft word!

ICF, please maintain the light grey background and change all light blue colored text and icons in the examples in action to a dark blue.

**Comment by jfc3-dol** [comment]
@adriennerlong,

The light gray background and dark blue text works.

**Comment by jfc3-dol** [comment]
@adriennerlong,

The color contrast has not been changed. The light blue text ( #1FAADB ) on the gray background ( #E0E0E0 ) has a color contrast ratio of 2.03:1 and needs to be 4,5:1 or higher.

The dark blue one shown above works.

**Comment by jfc3-dol** [comment]
Issue applies to both iOS and Android along with the website.

**Comment by adriennerlong** [comment]
From ICF: Fixed

**Comment by jfc3-dol** [comment]
There seems to be enough color contrast for the iOS version, so I'm going to remove the iOS label. I will check the Android and web version once they are available for me to test them.

**Comment by sanganalakshay** [comment]
should be fixed on both web and android too

**Comment by jfc3-dol** [comment]
Looked at the Android and web versions and the color contrast now passes.

Removing the Android and web label and closing the issue.
